packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -179,6 +179,7 @@ class Draggable<T extends Object> extends StatefulWidget {
     this.ignoringFeedbackPointer = true,
     this.rootOverlay = false,
     this.hitTestBehavior = HitTestBehavior.deferToChild,
+    this.buttonsEventFilter,
```

Comment:
We could just accept `PointerEvent event` instead of `int buttons`, which is fine. But might be too granular, as we are almost making `isPointerAllowed` a parameter.